src/components/NavButton/index.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { ReactNode } from "react";
+import { Button, ButtonProps } from "antd";
+import classNames from "classnames";
+
+import styles from "./style.css";
+
+export interface NavButtonProps extends ButtonProps {
+    titleText?: string;
+    buttonType?: string;
+    icon?: ReactNode;
+    clickHandler?: () => void;
+    isDisabled?: boolean;
+}
+
+const NavButton: React.FC<NavButtonProps> = ({
+    className,
+    titleText,
+    buttonType,
+    icon,
+    clickHandler,
+    isDisabled,
+    ...props
+}) => {
+    const buttonClassNames = classNames(
+        className,
+        styles.navButton,
+        buttonType && styles[buttonType],
```

Comment:
is there a default button type? you could set that so you don't need to have a conditional here

Reply:
Great idea, there is now!